manifests/install.pp

```diff
@@ -12,14 +12,32 @@
     ]:
       ensure => 'directory',
     }
+
```

Comment:
for style, can you compact all the vertical whitespace in the "windows" block to make it more obvious to the eye that these are all for windows, and then align arrows?